examples/sam/README.md

```diff
@@ -0,0 +1,125 @@
+# AWS Lambda Powertools for TypeScript examples in SAM
+
+This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
+
```

Comment:
Please add a paragraph/section [like this other one](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/examples/cdk/README.md?plain=1#L5) to put a disclaimer about potential costs that users might incur by deploying this example.